File: A0714db/param.js
/**
 * 아이디, 나이는 필수. 나이는 숫자만 입력 가능
 */

function chk(){
	var uid = addPer.uid.value;
	var age = addPer.age.value;
	
	if((uid == null) || (age == null)){
		alert("아이디와 나이는 필수 입력입니다.");
	} 
	else if(isNaN(age)) {
		alert("나이는 숫자만 입력하세요");
	}
	
	//비밀번호 체크
	var pw1 = addPer.pass1.value;
	var pw2 = addPer.pass2.value;
	
	if(pw1 != pw2){
	    alert("비밀번호가 틀렸습니다. ");
	}
	
	//전화번호
	var tel = addPer.tel.value;
	if(tel.indexOf("-") == -1){
		alert("'-'을 포함해서 입력하세요");
		addPer.tel.focus();
	}
}

File: A0714db/Param.java
package bean;

import java.io.IOException;
import java.io.PrintWriter;

import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

@WebServlet("/Param")
public class Param extends HttpServlet {
	private static final long serialVersionUID = 1L;
       
    public Param() {
        super();
    }

	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		//response.getWriter().append("Served at: ").append(request.getContextPath());
	}

	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		response.setContentType("text/html; charset=UTF-8");
		request.setCharacterEncoding("UTF-8");
		
		String uid = request.getParameter("uid");
		String age = request.getParameter("age");
		String pass1 = request.getParameter("pass1");
		String tel = request.getParameter("tel");
		
		//저장 정보 출력하기
		PrintWriter out = response.getWriter();
		out.print("<body>");
		out.print(uid +"님 반갑습니다." + "<br>");
		out.print("나이는 "+ age +"이고 전화번호는 "+ tel + "입니다. <br>");
		out.print("설정한 비밀번호: "+pass1 + "<br>");
		out.print("</body>");
		out.close();
		
	}

}
